src/App.js

```diff
@@ -20,13 +22,26 @@ function App() {
       }
     )
       .then((response) => response.json())
+
       .then((result) => {
-        setTodoList(result.records);
+        result.records.sort(function (objectA, objectB) {
+          if (objectA.fields.Title < objectB.fields.Title) {
```

Comment:
This is close! But I'm getting things unsorted when I run the app locally. See the screenshot!
![Screenshot 2023-04-24 at 8 46 22 AM](https://user-images.githubusercontent.com/5691923/234032333-ddf1719a-2b8c-4fb7-95a3-f9fe2fc01129.png)